go/service/account.go

```diff
@@ -260,3 +261,24 @@ func (h *AccountHandler) TimeTravelReset(ctx context.Context, arg keybase1.TimeT
 
 	return err
 }
+
+func (h *AccountHandler) GuessCurrentLocation(ctx context.Context, arg keybase1.GuessCurrentLocationArg) (string, error) {
+	mctx := libkb.NewMetaContext(ctx, h.G())
+	res, err := mctx.G().API.Get(mctx, libkb.APIArg{
+		Endpoint:       "account/location_suggest",
+		SessionType:    libkb.APISessionTypeNONE,
+		InitialTimeout: 5 * time.Second,
```

Comment:
15 seconds is kind of long for something that's optional, is that ok?

Reply:
i dont know what it should be, i just went with the gapu params

Reply:
where is it used? is it blocking anything?

Reply:
It blocks the following:
1) Signup, which has this as a call to "prepare" the next step.
2) Adding a new phone number in the accounts tab.

That being said if (2) already knows the current location it skips the call altogether.

Reply:
it blocks signup even though its only needed for phone #? idk, might be better to put it afterwards. may want to decrease total time in any case to 2-3 seconds since its only advisory. i'd ask @zapu though

Reply:
is there a spinner and the screen is blocked or can you still press skip in the GUI?

it's worth putting a long sleep in that RPC and testing how GUI reacts

Reply:
Fixed!